docs/pipelines_yaml_files.md

```diff
@@ -93,7 +93,17 @@ WGS: # Top of the pipeline (Library Prep)
 
 The other keys are detailed below.
 
-#### Filters
+#### pipeline_group
+
+This groups several Limber pipelines together that are part of the same real world pipeline.
+
+For instance, 'Heron-384 Tailed A V2' and 'Heron-384 Tailed B V2' - the split here is purely for technical reasons, to allow branching. In reality, they are both part of the Heron pipeline.
+
+Another example is when there are separate Limber pipelines for sequential stages. For instance, 'pWGS-384' (the library prep part) and 'pWGS-384 MX' (the multiplexing part). In reality, these are both part of the same pipeline, so they both have the pipeline group 'pWGS-384'.
+
+The pipeline group is used in the 'Work in progress' pages and the 'Pipelines overview' page.
```

Comment:
'and the 'Pipelines overview' page' - not yet, but will be after https://github.com/sanger/limber/issues/1558 is done.